dockstore-webservice/src/main/resources/migrations.1.4.0.xml

```diff
@@ -271,4 +271,15 @@
              UPDATE workflow SET giturl = 'git@github.com:' || substring(giturl, 16) where giturl like 'git@github.com/%'
         </sql>
     </changeSet>
+
+    <changeSet id="drop-toolname-null-constraint" author="agduncan">
+        <dropNotNullConstraint tableName="tool" columnName="toolname"></dropNotNullConstraint>
+
+        <dropDefaultValue tableName="tool" columnName="toolname"></dropDefaultValue>
+
+        <sql dbms="postgresql">
+            UPDATE tool SET toolname = NULL WHERE toolname = ''
```

Comment:
If nullable, set a constraint so that empty string is not allowed 
(i.e. only null or a non-empty string are valid) for both tool and workflow

Reply:
The constraint already existed for workflowname, but I added it for toolname.